packages/zoe/exported.js

```diff
@@ -2,5 +2,7 @@ import './src/contractFacet/types';
 import './src/zoeService/types';
 import './src/contractSupport/types';
 import './src/types';
+import './tools/types';
+import '@agoric/notifier/exports';
```

Comment:
hmm, can we make this match the other filenames? Not necessarily important for this PR, but maybe let's make an issue from this comment.

Reply:
Refs #1950